streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamImpl.java

```diff
@@ -760,8 +720,14 @@ public boolean test(final K1 key, final V1 value) {
                                         this.repartitionRequired);
     }
 
-
-
+    private <K, V> KeyValueMapper<K, V, String> createDefaultKeyValueMapper() {
+        return new KeyValueMapper<K, V, String>() {
```

Comment:
I think we should just create this as a field on the instance? I don't see any reason to create it everytime. Also makes the code cleaner